lib/elixir/lib/stream.ex

```diff
@@ -1404,6 +1401,22 @@ defmodule Stream do
     do: %{lazy | funs: [fun | funs], accs: [acc | accs], done: done}
   defp lazy(enum, acc, fun, done),
     do: %Stream{enum: enum, funs: [fun], accs: [acc], done: done}
+
+  defp do_interval({:first_time, interval}) do
```

Comment:
I just followed the same convention as the rest of the code, I personally *hate* `do_`.